igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/TravisBuildMonitor.groovy

```diff
@@ -74,6 +74,8 @@ class TravisBuildMonitor implements PollingMonitor{
 
     static final int NEW_BUILD_EVENT_THRESHOLD = 1
 
+    static final long BUILD_STARTED_AT_THRESHOLD = 30000L
```

Comment:
I think this might read better as `TimeUnit.SECONDS.toMillis(30)`, especially where the constant is added to the ms value of `cachedJobTTLDays`.